td-client.cpp

```diff
@@ -243,10 +243,21 @@ void PurpleTdClient::processUpdate(td::td_api::Object &update)
 void PurpleTdClient::processAuthorizationState(td::td_api::AuthorizationState &authState)
 {
     switch (authState.get_id()) {
-    case td::td_api::authorizationStateWaitEncryptionKey::ID:
+/*    case td::td_api::authorizationStateWaitEncryptionKey::ID:
         purple_debug_misc(config::pluginId, "Authorization state update: encriytion key requested\n");
         m_transceiver.sendQuery(td::td_api::make_object<td::td_api::checkDatabaseEncryptionKey>(""),
                                 &PurpleTdClient::authResponse);
+        break;*/
```

Comment:
Why introduce dead code? If this been removed, then remove the code. If we just don't support it, at least emit a useful information message.

Reply:
Fixed. Yes, looks like it has been removed.